sgrep/tests/python/equivalence_f_string_2.py

```diff
@@ -0,0 +1,27 @@
+def foo1():
+  # ERROR:
+  select = "select * "
+  query = f"{select} from foo"
+
+def foo2():
+  # ERROR:
+  select = "select * "
+  name = "foo"
+  query = f"{select} from foo " + f"where name={name}"
+
+def foo2a():
+  # TODO:
+  # OK:
+  select = "select * "
+  name = "foo"
+  query = f"{select} from foo where name={name}"
```

Comment:
Also it seems like this should match

Reply:
NM I'm silly

Reply:
From https://github.com/returntocorp/sgrep/pull/347#discussion_r401177492,  I use constant_propagation_and_evaluate_literal to determine if variable name is a literal. As the moment, the constant propagation in python does not seem to keep track of name as a literal